### PR TITLE
fix some typos in documentation

### PR DESCRIPTION
**Pull request by dongwenjuan**
Signed-off-by: dongwenjuan <dong.wenjuan@zte.com.cn>

### CODE REVIEW

File: Documentation/design/building-your-csv.md [hunk 1]
```diff
@@ -238,11 +238,11 @@ The metadata section contains general metadata around the name, version and othe
...
+**InstallModes**: A set of `InstallMode`s that tell OLM which `OperatorGroup`s an Operator can belong to. Belonging to an `OperatorGroup` means that OLM provides the set of targetted namespaces as an annotation on the Operator's CSV and any deployments defined therein. These deployments can then untilize [the Downward API](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api) to inject the list of namespaces into its container(s). An `InstallMode` consists of an `InstallModeType` field and a boolean `Supported` field. There are four `InstallModeTypes`:
```
[the remark this answers is not in the document]
Comment (dongwenjuan) [reply Documentation/design/building-your-csv.md]: @njhale  Sure, resolved.

**Comment by njhale** [comment]
Looks like this needs a rebase.

**Comment by jpeeler** [comment]
/retest
Feel free to update with the other correction Nick suggested earlier.

**Comment by ecordell** [comment]
/lgtm

we can follow up on the other typo

File: Documentation/design/dependency-resolution.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Dependency Resolution and Upgrades
 
-OLM manages the dependency resolution and upgrade lifecycle of running operators. In many ways, thes problems OLM faces are similar to other OS pacakge managers like `apt`/`dkpg` and `yum`/`rpm`.
+OLM manages the dependency resolution and upgrade lifecycle of running operators. In many ways, these problems OLM faces are similar to other OS pacakge managers like `apt`/`dkpg` and `yum`/`rpm`.
```
Comment (njhale) [inline Documentation/design/dependency-resolution.md]: > In many ways, these problems OLM faces are similar to other OS pacakge managers like `apt`/`dkpg` and `yum`/`rpm`.

In many ways, the problems OLM faces are are similar to those faced by other OS package managers like `apt`/`dpkg` and `yum`/`rpm`.

**Comment by ecordell** [comment]
we need to do a full review of our docs, so closing this for now.

**Pull request closed without merging**